fronts-client/src/components/feed/RecipeSearchContainer.tsx

```diff
@@ -56,11 +67,21 @@ export const RecipeSearchContainer = ({ rightHandContainer }: Props) => {
     chefSelectors.selectLastFetchOrder(state)
   );
 
+  const [page, setPage] = useState(1);
+
   const debouncedRunSearch = debounce(() => runSearch(), 750);
 
   useEffect(() => {
-    debouncedRunSearch();
-  }, [selectedOption, searchText]);
+    if (page > 1 && searchText.length === 0) runSearch(page);
+    else debouncedRunSearch();
+  }, [selectedOption, searchText, page]);
+
+  //const chefsPagination = IPagination | null;
```

Comment:
Could we remove this comment? I don’t think it adds anything, unless I misunderstand